usaspending_api/etl/management/sql/dev3627_compare_dabs_row_counts.sql

```diff
@@ -0,0 +1,126 @@
+WITH file_a AS (
+    SELECT submission_id, COUNT(aab.*) AS count
+    FROM submission_attributes sa
+    LEFT OUTER JOIN appropriation_account_balances AS aab USING (submission_id)
+    GROUP BY submission_id
+), file_b AS (
+    SELECT submission_id, COUNT(fapaoc.*) AS count
+    FROM submission_attributes sa
+    LEFT OUTER JOIN financial_accounts_by_program_activity_object_class AS fapaoc USING (submission_id)
+    GROUP BY submission_id
+), file_c AS (
+    SELECT sa.submission_id, COUNT(faba.*) AS count
+    FROM submission_attributes sa
+    LEFT OUTER JOIN financial_accounts_by_awards faba USING (submission_id)
```

Comment:
Really small, but would be nice to have `AS` in this line as well for consistency.

Reply:
Done, thanks